model/project.py

```diff
@@ -111,90 +111,112 @@ def __str__(self):
         """Return all fields of this project, JSON format."""
         return str(self.__dict__)
 
-    def get_project_id(self):
+    @property
```

Comment:
We can probably also remove most of these `@property` declarations and expose the fields directly, and only add `@property` back if we actually have to control what happens when that property is retrieved, right? That would save us quite a bit of code.

Reply:
Yes. That would also work. Will do.